### ISSUE TITLE
[Feature Request]Click 'Browse Data' opens selected Table in 'Database Structure'

### ISSUE THREAD

**Issue by sky5walk** [sqlitebrowser/sqlitebrowser#1643]
Describe the new feature
--------------------------
NOW: (The use model I cannot kick)
Open DB4S.
'Database Structure' appears 1st.
Left-mouse click a Table in that view. (I rarely right-mouse click)
Then left-mouse click 'Browse Data', but it opens the 1st Table in that drop-down list?
Frustratingly, I return to the drop-down list to select my desired Table.
(Think about creating a Feature Request to end the extra clicks :( )
Repeat.

AFTER: (Feature Request implemented)
Open DB4S.
'Database Structure' appears 1st.
Left-mouse click a Table in that view.
(DB4S makes this the selected Table in the 'Database Structure' Tab.)
Then left-mouse click 'Browse Data', and it opens the Table selected in 'Database Structure'.
(Give a hoorah for such efficiency :) )
Repeat.

Additional info
---------------
[A similiar request was closed without the feature above implemented.](https://github.com/sqlitebrowser/sqlitebrowser/issues/246)
[Somewhat related bug closed.](https://github.com/sqlitebrowser/sqlitebrowser/issues/757)

**Comment by chrisjlocke** [comment]
> A similiar request was closed without the feature above implemented

It was implemented, but in the right click menu which you don't want to use.

**Comment by sky5walk** [comment]
Yes, I did mention that, but I still prefer the accuracy of 2 left-mouse clicks.
Right-mouse clicking requires scrolling a smallish popup and is error prone.
